diff --git a/tools/extras/install_openblas.sh b/tools/extras/install_openblas.sh
index ce0fdf7..d134c2f 100755
--- a/tools/extras/install_openblas.sh
+++ b/tools/extras/install_openblas.sh
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-OPENBLAS_VERSION=0.3.13
+OPENBLAS_VERSION=0.3.20
 
 WGET=${WGET:-wget}
 
@@ -30,7 +30,7 @@ else
 fi
 
 tar xzf $tarball
-mv xianyi-OpenBLAS-* OpenBLAS
+mv OpenMathLib-OpenBLAS-* OpenBLAS
 
 make PREFIX=$(pwd)/OpenBLAS/install USE_LOCKING=1 USE_THREAD=0 -C OpenBLAS all install
 if [ $? -eq 0 ]; then
